MARKERS.md

```diff
@@ -47,15 +47,15 @@ Single line changes:
 
 ```yml
 <Rest of the unchanged file above...>
-[Your code here. This can be entirely original, or modified from the original file] # Parkstation-[TitleOfFeature] - Optional clarification of why this change was done
+[Your code here. This can be entirely original, or modified from the original file] # Parkstation-[TitleOfFeature] E Optional clarification of why this change was done
```

Comment:
```suggestion
[Your code here. This can be entirely original, or modified from the original file] # Parkstation-[TitleOfFeature] # Optional clarification of why this change was done
```

E